src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -71,6 +71,29 @@ public function getConfigTreeBuilder()
             ->arrayNode('serializer')
                 ->{interface_exists(SerializerInterface::class) ? 'canBeDisabled' : 'canBeEnabled'}()
             ->end()
+            ->arrayNode('translation_extractor')
+                ->canBeEnabled()
+                ->fixXmlConfig('path')
+                ->children()
+                    ->arrayNode('paths')
+                        ->example(['%kernel.project_dir%/src/Enum' => 'App\Enum'])
+                        ->useAttributeAsKey('name')
+                        ->scalarPrototype()->cannotBeEmpty()->end()
+                    ->end()
+                    ->scalarNode('domain')
+                        ->defaultValue('elao_enum')
```

Comment:
What would you think about keeping the default `messages` domain here?
At least on own projects, I never used a dedicated trans domain for enum nor for specific vendor related translations. It's most often driven by the app scopes.